test/evergreen.yml

```diff
@@ -143,6 +151,26 @@ tasks:
 
             ${test_env_vars|} ${make_command|make} VERBOSE=1 check ${smp_command|} 2>&1
 
+  - name: make-check-msan-test
+    depends_on:
+      - name: compile-msan
+    commands:
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: compile-msan
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ MSAN_OPTIONS=abort_on_error=1:disable_coredump=0 MSAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer CFLAGS=-fsanitize=memory
+          posix_configure_flags: --enable-silent-rules --enable-strict --enable-diagnostic --disable-static
```

Comment:
It would be great to abstract this section of code e.g. into a diff func, considering it's called twice in this change.